sources/driverdog/src/main.rs

```diff
@@ -281,21 +372,32 @@ fn load_modules_sets(
     modules_sets: &HashMap<String, DriverConfig>,
     target: Option<String>,
 ) -> Result<()> {
-    // Update the modules.dep before we attempt to load kernel modules
-    let args: Vec<String> = Vec::new();
-    command(DEPMOD_BIN_PATH, args)?;
-    info!("Updated modules dependencies");
-
     // If the target module set was given, load the kernel modules in it
     if let Some(target) = target {
         let driver_config = modules_sets
             .get(&target)
             .context(error::MissingModuleSetSnafu { target })?;
+
+        // If the configuration specifies a depmod file, include it in the args
+        // Only valid if targetting a specific module set
+        let mut args: Vec<String> = Vec::new();
+        if let Some(depmod_file) = &driver_config.depmod_file {
+            args.push(format!("--config={}", depmod_file));
```

Comment:
Do we need to include the custom depmod config change here? Is it needed for the eventual solution?